blpapi.ts

```diff
@@ -0,0 +1,728 @@
+/// <reference path="./typings/tsd.d.ts" />
+
+'use strict';
+
+import assert = require('assert');
+import events = require('events');
+import util = require('util');
+
+import Promise = require('bluebird');
+import debug = require('debug');
+import _ = require('lodash');
+
+// Somewhat hacky but needed to import the 'blpapijs' module.
+import path = require('path');
+process.env.NODE_PATH = path.join(__dirname, 'build/Release');
+/* tslint:disable:no-var-requires */
+require('module').Module._initPaths();
```

Comment:
I guess you can even import `module` beforehand and call the `_initPaths` here, so you can avoid the tslint error.